pkg/apis/velero/v1/backup.go

```diff
@@ -101,9 +101,15 @@ type BackupSpec struct {
 	// DefaultVolumesToRestic specifies whether restic should be used to take a
 	// backup of all pod volumes by default.
 	// +optional
-	// + nullable
+	// +nullable
```

Comment:
How about adding the `deprecated` marker to indicate that the property will be removed in the next version of CRD? https://book.kubebuilder.io/reference/markers/crd.html

Reply:
As the discussion, we will not use the `deprecated` marker for the current case. The reason is as below:
We will not change the CRD's version(as the conclusion here). On the other hand, the above `deprecated` marker is for the CRD spec wides, that is to mark a deprecated CRD; while our requirement is to mark a single field for a deprecation in future version (it is just a warning), so it is not suitable to our requirement.